Modules/Package.swift

```diff
@@ -1,20 +1,35 @@
-// swift-tools-version: 6.0
+// swift-tools-version: 5.10
```

Comment:
Downgraded to the latest 5.x because I was getting build issues. 

The previous TestKit implementation used 5.5 (see the deleted `TestKit/Package.swift`) so this is also a consistency change. We can consider bumping the Swift version across both Woo app and its modules once the migration is completed.